src/recipe.js

```diff
@@ -21,12 +21,28 @@ class Recipe {
   }
 
   getInstructions() {
-    let instructions = this.instructions.map(recipe => {
-      return recipe.instruction
-    });
-    return instructions
+    return this.instructions
```

Comment:
Can't believe we didn't catch this right away, looks so much better now!!

Reply:
Oh my, that makes perfect sense!